Cython/Compiler/FusedNode.py

```diff
@@ -686,31 +758,64 @@ def __pyx_fused_cpdef(signatures, args, kwargs, defaults):
             self._buffer_declarations(pyx_code, decl_code, all_buffer_types, pythran_types)
             env.use_utility_code(Code.UtilityCode.load_cached("Import", "ImportExport.c"))
             env.use_utility_code(Code.UtilityCode.load_cached("ImportNumPyArray", "ImportExport.c"))
-
+        
+        context.update(
+            global_sigindex_name = "'__cython__fusedcpdef__sigindices'",
```

Comment:
Is this okay? If it is, is there a convention I should follow for internal names like this?

Reply:
There's a file Cython/Compiler/Naming.py that defines all the internal names. It might be better to add this constant to Naming.py (so if it ever gets changed you only have so change it in one place)?

Reply:
> if it ever gets changed you only have so change it in one place

That, and it makes it easier to avoid accidental global naming collisions by defining all prefixes in one place.

Reply:
Looking at that file, it seems to be exclusively for C names. Would this name fit in there given that it's dynamically assigned with a call to `globals()` at run time?